app/src/main/res/values-night/colors.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <color name="primary">@color/periwinkle</color>
```

Comment:
periwinkle? :D

Reply:
hahahaha yeah, when you export the colours from zeplin you get the "friendly name" for that colour. I though periwinkle is funny enough so I left it 😄 

PS. It's the "light purple"